README.md

```diff
@@ -16,7 +16,7 @@ This project includes pytest unit test and llvm-style filecheck tests. They can
 
 ```
 # Executes pytests which are located in tests/
-pytest
+python -m pytest
```

Comment:
I am surprised this causes a difference. Are you sure this change is needed, and the change below is not sufficient by itself. If both are needed, we should probably change the documentation as suggested here.

Reply:
This isn't necessary but my impression is that it's the recommended best practice. Python versions and tools can go out of sync for reasons that are still unclear to me, and my local `pytest` was not the same as `python  -m pytest`. This change guarantees that they'll be the same, even if it's a bit more verbose

Reply:
Found a blog post that explains that there are differences, my understanding is that the `-m` is slightly preferred in this case, but I'm not 100% sure still.
https://blog.ganssle.io/articles/2019/08/test-as-installed.html

Reply:
Interesting. I then do not have any preference.

Reply:
Re-reading the post, looks like it might be better without the `-m`, I'll remove this change